treeherder/model/derived/jobs.py

```diff
@@ -108,9 +108,6 @@ class JobsModel(TreeherderModelBase):
         "jobs.deletes.cycle_job_note",
         "jobs.deletes.cycle_bug_job_map",
         "jobs.deletes.cycle_job",
-        "jobs.deletes.cycle_revision",
-        "jobs.deletes.cycle_revision_map",
-        "jobs.deletes.cycle_result_set"
     ]
```

Comment:
Hi @edmorley , I must say it's wierd because I had deleted these lines at beginning. And I address line line(https://github.com/mozilla/treeherder/pull/1146/files#diff-a03f6e7c1747a7ef236e211422cc7c5eR548) to jobs_to_cycle in my last commit. But when I click into the (raw file) link, it seems like nothing changed. Is that a bug or somethings? or I misunderstand about your comment?